app/server/tests/integration/errors.test.ts

```diff
@@ -40,22 +42,23 @@ describe("Error handling", () => {
         expect(amrRes.status).toBe(200);
 
         // 3. Run poppunk and wait til it finishes
-        testSample.projectId = projectId;
-        const poppunkRes = await post(`poppunk`, testSample, connectionCookie);
+        const fakeProjectHash = `${projectId}ABC`;
+        const projectData = testSample(fakeProjectHash, projectId);
+        const poppunkRes = await post(`poppunk`, projectData, connectionCookie);
```

Comment:
So I think part of the problem was that we weren't using a unique project hash every time we ran poppunk, so I've changed `testSample` from an object to a function, and making a new hash for each new project id...